src/EPR.Payment.Facade.Common.UnitTests/RestServices/HttpOnlinePaymentsServiceTests.cs

```diff
@@ -34,15 +34,15 @@ public void Initialize()
             var config = new Service
             {
                 Url = "https://example.com",
-                EndPointName = "payments",
+                EndPointName = "onlinepayments",
```

Comment:
Should be online-payment

Reply:
it is EndPointName = "online-payments", now